### PR TITLE
Stop using Prebid labels in the codebase, move logic to functions instead 

### PR DESCRIPTION
**Pull request by katebee**
#### 👉 Removes Prebid Labels from the codebase

Currently some Prebid code uses the Prebid label system to specify conditions and other code specifies conditions in native JS. 

It would be best to remove all Prebid labels due to weaknesses in label system:

* Prebid labels have constraints in the way they can be combined: they can only be combined in a single AND or OR clause.
* Labels require all the logic to build parameters for bidders that may subsequently never be used.
* There's some similar logic in setting up config params for bids as deciding whether to include bids in auction or not, but can't easily reuse labels in both places.

#### 👉 Removes some custom types associated with Prebid 

Custom types that are only in use in one place add a layer of abstraction and can make type inference more difficult, or hide the type entirely in some IDEs.

#### 👉 Update tests due to mocked utils, and tinker with setup and teardown steps

Changing the logic and adding some more util functions to determine if the bids should include Improve Digital and Xaxis has changed the outcome of some tests (due to mocking utils). 

Adjustments to beforeEach and afterEach are an attempt to solve the flakey `shouldIncludeOzone` tests so they can start running again.  😁 

Would be great to cease mocking Utils or change up how we are testing big-config to cover a bit more of the integration and record the business logic as test statements... However I think that is a large enough task to tackle another time. 

## Screenshots

But what is PrebidBanner Flow? Tell me that!

screen shot 2018-10-29 at 12 11 18

Oh, okay thanks!

screen shot 2018-10-29 at 13 17 25

😅 🌈 🦄 

### Does this affect other platforms?

Nope

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

Nope

### Does this change break ad-free?

Nope

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/prebid/utils.js [hunk 1]
```diff
@@ -128,6 +128,20 @@ export const shouldIncludePangaea = (): boolean =>
...
+        return hasFirstLook;
+    }
+    return false;
+};
+
+export const shouldIncludeImproveDigital = (): boolean => {
+    const edition: ?string = config.get('page.edition');
+    return edition === 'UK' || edition === 'INT';
+};
+
```
Comment (kelvin-chappell) [inline static/src/javascripts/projects/commercial/modules/prebid/utils.js]: I think these could be geotargeted, but probably best to do in a separate PR.

File: static/src/javascripts/projects/commercial/modules/prebid/slot-config.js [hunk 2]
```diff
@@ -6,51 +6,41 @@ export const slots: PrebidSlot[] = [
...
         key: 'mostpop',
         sizes: [[300, 250]],
-        labelAny: ['mobile', 'tablet', 'desktop'],
     },
     {
         key: 'comments',
         sizes: [[300, 250]],
-        labelAny: ['desktop'],
     },
 ];
```
Comment (kelvin-chappell) [inline static/src/javascripts/projects/commercial/modules/prebid/slot-config.js]: Will need to do these size restrictions in conditions as well.  So that `top-above-nav` asks for different sizes according to breakpoint etc.

**Review by kelvin-chappell — CHANGES_REQUESTED** [review]
Looks good but I think sizes will need to be set according to context as well.

**Review by kelvin-chappell — APPROVED** [review]

File: static/src/javascripts/projects/commercial/modules/prebid/price-config.js [hunk 3]
```diff
@@ -1,6 +1,13 @@
 // @flow strict
 
-import type { PrebidPriceGranularity } from 'commercial/modules/prebid/types';
+export type PrebidPriceGranularity = {
```
Comment (katebee) [inline static/src/javascripts/projects/commercial/modules/prebid/price-config.js]: This probably doesn't need to be an export type since we do not rely on the type definition anywhere else in the codebase... 🤔

File: static/src/javascripts/projects/commercial/modules/prebid/utils.js (hunk 1 above)
Comment (katebee) [reply static/src/javascripts/projects/commercial/modules/prebid/utils.js]: Agreed! Ideally want to remove all references to config.page.edition from Prebid 😁

Added this as a card here: https://trello.com/c/PQixSSBg
